internal/utils/csi.go

```diff
@@ -326,7 +326,9 @@ var PoolTimeContainerArg = "--polltime=60s"
 var ExtraCreateMetadataContainerArg = "--extra-create-metadata=true"
 var PreventVolumeModeConversionContainerArg = "--prevent-volume-mode-conversion=true"
 var HonorPVReclaimPolicyContainerArg = "--feature-gates=HonorPVReclaimPolicy=true"
-var TopologyContainerArg = "--feature-gates=Topology=true"
+
```

Comment:
```suggestion
```

Reply:
it's getting auto added, not my doing, might need to add `nofmt` tag.

Reply:
ok, resolving